src/main/java/org/jabref/gui/maintable/OpenUrlAction.java

```diff
@@ -62,6 +62,10 @@ public void execute() {
                 link = entry.getField(StandardField.URL);
                 field = StandardField.URL;
             }
+            if (entry.hasField(StandardField.ISBN)) {
+                link = Optional.of("https://openlibrary.org/isbn/" + entry.getField(StandardField.ISBN).get());
```

Comment:
Having a non-trivial string explicitly in the middle of a code directly leads to the question: Isn't it possible that this is a constant. Then I looked, how `JabRefDesktop.openExternalViewer` works.

I think, the petter place to map an ISBN to an URL would be htere. Maybe, even introuce `openIsbn` (analogous to `openDoi`).

Reply:
addressed